lang/nl.json

```diff
@@ -0,0 +1,18 @@
+{
+    ":count of the requested quantity will be backordered": ":count van de aangevraagde hoeveelheid zal worden nageleverd",
+    "Apply discount code": "Kortingscode toepassen",
+    "Disclaimer": "Disclaimer",
+    "Discount": "Korting",
+    "Enter code": "Code invoeren",
+    "Free and easy returns": "Gratis en eenvoudige retouren",
+    "Free shipping from €65": "Gratis verzending vanaf €65",
+    "Ordered before 7:00 PM, delivered tomorrow": "Voor 19:00 uur besteld, morgen geleverd",
```

Comment:
These feels a little hardcoded, but i see that it's from an example usp correct?

Reply:
Yes, I've removed them since they should be overwritten